_data/hosting.yml

```diff
@@ -451,6 +451,13 @@ websites:
       - totp
     doc: https://support.stackpath.com/hc/en-us/articles/360022992631
 
+  - name: switchplus
+    url: https://switchplus.ch/
+    img: switchplus.png
+    tfa:
+      - sms
+    doc: https://switchie.ch/2fa/
```

Comment:
This link redirects to https://switchplus.ch/
After a brief search, I couldn't find any documentation.

Reply:
The link works for me... Strange. How about this one in English?
https://switchie.ch/en/2-factor-authentication-2fa/

Reply:
For me, that link also redirects to https://switchplus.ch/
I can read the contents preview [using a search engine](https://duckduckgo.com/?q=2+factor+authentication+site%3Aswitchie.ch&t=ffab&ia=web).

Reply:
I can reproduce that with a VPN connection via North America. I'd say we use the link to the English page anyway.